sfs/mono/nfchoa.py

```diff
@@ -0,0 +1,259 @@
+"""Compute driving functions for various systems.
+
+.. include:: math-definitions.rst
+
+.. plot::
+    :context: reset
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import sfs
+
+    plt.rcParams['figure.figsize'] = 6, 6
+
+    xs = -1.5, 1.5, 0
+    xs_focused = -0.5, 0.5, 0
+    # normal vector for plane wave:
+    npw = sfs.util.direction_vector(np.radians(-45))
+    # normal vector for focused source:
+    ns_focused = sfs.util.direction_vector(np.radians(-45))
```

Comment:
`ns_focused` is unused.

Reply:
23c4f8e575f3d7a63283412ce7de1e43d55d347d